package.json

```diff
@@ -105,6 +105,7 @@
         "build": "rm -rf lib/build && npx rollup -c",
         "watch": "npx rollup -cw",
         "watch-mac": "chokidar lib/ts/* -c 'npx rollup -c'",
+        "watch-mac-2": "chokidar lib/ts/* -c 'npx rollup -c && cp -r lib recipe ui index.d.ts index.js package.json rollup.config.mjs webJsInterfaceSupported.json ../../supertokens-demo/frontend/node_modules/supertokens-auth-react/ && rm -rf ../../supertokens-demo/frontend/node_modules/.cache'",
```

Comment:
what is this for?

Reply:
removed it. This was not supposed to be committed.